examples/ios/.bazelrc

```diff
@@ -0,0 +1,3 @@
+build --apple_crosstool_top=@local_config_apple_cc//:toolchain
```

Comment:
Is there any documentation on why these are needed? Perhaps some github issue that refers to these?

Reply:
the setup doc talks about the general usage https://github.com/bazelbuild/apple_support/

but the gist is until the incompatible flag for CC toolchain resolution ships this is how you have to specify anything besides the default cc toolchain. in the future once that flag is flipped (and the behavior is ironed out in general) this entire file can be tossed because the setup will happen in the WORKSPACE and that will be enough

Reply:
Could you add that link with this context there? I'm planning on working on https://github.com/bazelbuild/rules_rust/issues/1839 and think it'd be nice to have the ios examples act as examples but to do that they need to be well documented. Every little bit helps.

Reply:
done